src/layout/PersonLookup/PersonLookupComponent.tsx

```diff
@@ -219,7 +220,7 @@ export function PersonLookupComponent({ node, overrideDisplay }: PropsFromGeneri
           required={required}
           readOnly={hasSuccessfullyFetched}
           error={
-            (nameErrors?.length && <Lang id={nameErrors.join(' ')} />) ||
+            (nameError?.length && <Lang id={nameError} />) ||
```

Comment:
Siden det ikke er en liste lenger kan du bare sjekke om den er truthy (måtte ha length for å sjekke at det var elementer i lista tidligere, siden en tom liste er truthy) 😊 
```suggestion
            (nameError && <Lang id={nameError} />) ||
```